Airport Ticket Booking/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlTypes;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static Airport_Ticket_Booking.Common;
+
+namespace Airport_Ticket_Booking
```

Comment:
I have a stretch goal for you here. What design pattern would be appropriate here? Can you come up with one and how to implement it?